docs/en/deploy/Requirements.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: global
+title: Alluxio Software Requirements
+nickname: Software Requirements
+group: Deploying Alluxio
+priority: 9
+---
+
+{:toc}
+
+## General Requirements
+
+There are base requirements for cluster nodes:
+
+* Cluster nodes should be running one of the following supported operating systems:
+  * MacOS 10.10 or later
+  * CentOS - 6.8 or 7
+  * RHEL - 7.x
+  * Ubuntu - 16.04
+* Alluxio is a Java application which requires version 8 of the JRE:
+  * Java JRE 8
+* Alluxio works on IPv4 networks only.
+* Allow the following ports and protocols:
+  * Inbound TCP 22 - ssh as a user to install Alluxio components across specified nodes.
+
+## Master Requirements
+
+There are Alluxio-specific requirements for cluster nodes running the master process.
+
+Note that these are bare minimums to run the software. Running Alluxio at scale and under high load
+will increase these requirements.
+
+* Minimum 4 GB disk space
+* Minimum 4 GB memory
+* Minimum 4 CPU cores
+* Allow the following ports and protocols:
+  * Inbound TCP 19200 - The Alluxio master's default port for internal leader election
+  * Inbound TCP 19998 - The Alluxio master's default RPC port
+  * Inbound TCP 19999 - The Alluxio master's default web UI port. Accessible at `http://<master-hostname>:19999`
+  * Inbound TCP 20003 - The Alluxio job master's default port for internal leader election
+
+## Worker Requirements
+
+There are Alluxio-specific requirements for cluster nodes running the worker process:
+
+* minimum 1 GB disk space
+* minimum 1 GB memory
+* minimum 2 CPU cores
+* Allow the following ports and protocols:
+  * Inbound TCP 29999 - The Alluxio worker's default RPC and data transfer port
+  * Inbound TCP 30000 - The Alluxio worker's default web UI port. Accessible at `http://<worker-hostname>:30000` in your browser
+  * Inbound TCP 30001 - The Alluxio job worker's default RPC port
+  * Inbound TCP 30002 - The Alluxio job worker's default data transfer port
+  * Inbound TCP 30003 - The Alluxio job worker's default web UI port.  Accessible at `http://<worker-hostname>:30003` in your browser
+
+### RAMFS
```

Comment:
This section is a bit misleading because Alluxio does not require a ramfs

Reply:
Would it make sense to have this as an `optional` section in the requirements page? Or would you rather remove it completely?

Reply:
+1 for Optional